cmd/delete.go

```diff
@@ -22,15 +23,15 @@ var deleteCmd = &cobra.Command{
 
 func deleteGenesis(cmd *cobra.Command, args []string) error {
 	// TODO sanitize this input
-	genesis := filepath.Join(baseDir, args[0]+genesis_suffix)
-	sidecar := filepath.Join(baseDir, args[0]+sidecar_suffix)
+	genesis := filepath.Join(app.GetBaseDir(), args[0]+constants.Genesis_suffix)
```

Comment:
Can't we use the functions in `app` for this?

Reply:
I guess we can